ui/branding/semantic-ui/less/default/modules/dropdown.overrides

```diff
@@ -0,0 +1,7 @@
+// unreasonable min width of 14em for the drop down,
+// breaks the layout on small screens and protrudes outside of the container
+@media all and (max-width: @tabletBreakpoint) {
+  .ui.dropdown {
+    min-width: 0 !important;
```

Comment:
Setting it to zero might also break ui, showing nothing instead. We should probably set it to a reasonable min value suitable for small screens

Reply:
@mirekys Actually, it works quite nicely, as the dropdown simply wraps around the content (text inside). As example:

Big screen 

![image](https://github.com/Narodni-repozitar/nr-docs/assets/34883098/50efc0f2-964b-49cb-93ec-4fd4ccdf1261)


Smaller screen 

![image](https://github.com/Narodni-repozitar/nr-docs/assets/34883098/e52dbaa0-f6f5-4738-a744-807ff56680e9)

I find it maybe better to just let css do its thing, instead of arbitrarily guessing some size. What do you think?

Reply:
Agreed, in that case you can leave it as it is :)

Reply:
@mirekys thx